storybook/index.html

```diff
@@ -54,6 +54,26 @@ <h2 class="sbHeading">Modules</h2>
                     <a class="btnLink btnLink-red btnLink-large">Button</a>
                 </section>
 
+                <section class="demo" data-module-name="event">
+                    <div class="box event">
```

Comment:
with SMACSS you don't put module classes side-by-side, although it wouldn't be bad to design the box to be "mixed in." those are also called "utility classes": https://css-irl.info/a-year-of-utility-classes/

(but I was hoping to avoid utility classes unless there wasn't an alternative, because it's easy to go overboard with them and they're conceptually not much better than inline styles)

so anyway, just `<div class="box">` should be fine for the outer div. e.g.:

```css
<div class="box">
  <div class="anotherModule">
    ...
```